RTL_testcases/Raptor_examples/counter_vhdl/constraints.sdc

```diff
@@ -1,4 +1,7 @@
-# SDC file example
+# Only pin out one counter bit as an example. 
+set_pin_loc {counter[0]} HP_1_1_0N
 
-# Setting a clock frequency of 200 MHz (5nS period)
-create_clock -period 5 clock0
```

Comment:
@AYYAZmayo , please add again the create_clock, we need to add SDC constraints to all designs, not remove them.

Reply:
@alaindargelas Ok will do that. I mimic the tests present in Raptor example. Will revert them